### DISCUSSION TITLE
Question about incorporating payer reimbursement policy detail (for example, multi-procedure discounting rules)

### DISCUSSION THREAD

**Discussion by foustsWCH** [CMSgov/hospital-price-transparency#44] [Q&A] [answered]
Can you give some clarification on to what extent _payer reimbursement policies_ need to be reflected in the machine-readable file? Payer reimbursement policies are typically publicly available and would apply to all providers unless language in the provider's Agreement with the payer specifically supersedes the policy.  Does the machine-readable file have to explicitly address all of these policies and whether these policies do/do not apply to a particular charge or is that level of detail unnecessary?  I am trying to understand if the file always needs to explicitly state these or not.   Payers often have many reimbursement policies with frequent updates and they vary in complexity.

To give a concrete example - it is common for payers to have multi-procedure discounting for surgical procedures, imaging, therapy, etc.    For these types of policies, it would be typical that the first service is paid 100% of the contracted rate, the second might be only 50% of the contracted rate, etc. 

- We have a payer contract where our Agreement with the payer specifically dictates that multi-procedure discounting does NOT apply for surgical procedures, so the payer reimbursement policy does not apply. All procedures are reimbursed at 100% of the negotiated contract rate.  So, with that example, for Procedure CPT XXXXX, is it acceptable to just list the contracted rate in the MRF (for example, CPT XXXXX = $1000 fixed rate), or do we need to specifically indicate somewhere that multi-procedure discounting rules do not apply and that even if the procedure is not the primary procedure it is still reimbursed at 100% of the negotiated rate?  
-Would this be an appropriate way to display it in the CSV "wide" format or is the comment unnecessary:

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

code |   | negotiated_dollar | negotiated_percentage | negotiated_algorithm | estimated_amount | methodology | additional_payer_notes
-- | -- | -- | -- | -- | -- | -- | --
XXXXX | CPT | 1000 |   |   |   | fee schedule | Reimbursed at 100% of negotiated rate when primary or subsequent procedure; Not subject to multi-procedure discounting

</body>

</html>

- The same contract does not make any restrictions on multi-procedure discounting for imaging, so the payer reimbursement policy applies even though the language is only in the payer policy and not directly in our contract document itself.  Do we need to list the payment reduction algorithm (100% if primary service; 50% for subsequent imaging services on same encounter) even though it is publicly available and would apply to all of the payer's contracted providers unless they've intentionally negotiated it out of their contract?   So if for example CPT 71250 is $500 (so paid $500 if primary, $250 if subsequent, etc.), how would this be displayed in the "wide" format?  Is it two separate rows or is it an algorithm?  OR, is this level of detail not required at all?

Should it be displayed as an algorithm where the primary rate is listed as a negotiated dollar, but an "estimated amount" reflects the typical reimbursement (reflecting the "weighted average" of how often this is reimbursed as primary vs subsequent)?
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

code |   | negotiated_dollar | negotiated_percentage | negotiated_algorithm | estimated_amount | methodology | additional_payer_notes
-- | -- | -- | -- | -- | -- | -- | --
71250 | CPT | 500 |   | Subject   to multi-procedure discounting (100% primary/50% subsequent) when multiple  imaging services are provided in the same encounter | $368 | fee schedule |  

</body>

</html>

Or is it more appropriate as two rows, one listing the rate if it's the primary imaging procedure and one rate if it's secondary:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

code |   | negotiated_dollar | negotiated_percentage | negotiated_algorithm | estimated_amount | methodology | additional_payer_notes
-- | -- | -- | -- | -- | -- | -- | --
71250 | CPT | 500 |   |   |   | fee schedule | When   primary imaging procedure
71250 | CPT | 250 |   |   |   | fee schedule | When   subsequent imaging procedure on same encounter as another imaging service

</body>

</html>

Thank you for clarifying both this example and my broader question.

**Comment by RidenourK** [comment, marked as answer]
@foustsWCH Thank you for your question. 

As explained in the CY2024 OPPS/ASC final rule, hospitals use different methods to establish standard charges for items and services, resulting in charge/item and charge/service combinations that are often unique to that hospital.  Therefore, although CMS has recently taken steps toward standardization, each hospital will continue to have some discretion related to how it chooses to encode its standard charge information (including information related to payer-specific negotiated charges) within its machine-readable file, so long as the file conforms to the CMS template layouts and data specifications as described at [45 CFR 180.50(c)(2)](https://www.ecfr.gov/current/title-45/part-180#p-180.50(c)(2)).  We note that, in addition to the examples you provided in your inquiry, the “general_contract_provisions” data element is an optional data element that a hospital may use to encode payer contract provisions (such as “stop-loss” or “lesser than” provision) that are applicable to a wide range of items and services.
